lib/pure/httpclient.nim

```diff
@@ -260,7 +260,7 @@ proc contentLength*(response: Response | AsyncResponse): int =
   ## A `ValueError` exception will be raised if the value is not an integer.
   var contentLengthHeader = response.headers.getOrDefault("Content-Length")
   result = contentLengthHeader.parseInt()
-  doAssert(result >= 0 and result <= high(int32))
+  doAssert(result >= 0 and result <= high(BiggestInt))
```

Comment:
```suggestion
  doAssert result >= 0
```
I don't think an `int` (which is what `result` is) can ever be bigger than `high(BiggestInt)`(there are a lot of places that assume that `int` is always 32 or 64 bits and `BiggestInt` is currently an alias for `int64`).

That being said, there was probably a reason to limit the length, but I can't comment on that.